### ISSUE TITLE
Search Results or No Results Found Message Not Announced with Screen Reader: A11y_ Visual Studio Code Services_Open Source AI code editor screen _Status Message.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-docs#9314]
### GitHub Tags: 
#A11yTCS; #A11yMAS; #A11ySev2; #A11yWCAG2.1; #DesktopWeb; #ChromiumEdge; #Win11; #Visual Studio Code Services; #E2E_Visual Studio Code Services_Web_Jan26; #WCAG4.1.3; #Status Message; #ES Chat;   
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification." 
Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2feng.ms%2fdocs%2fcoreai%2fdevdiv%2fone-engineering-system-1es%2f1es-docs%2fes-chat%2findex&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221175207%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=ju8Raw6mIhT5G3gWbZSkS0CAN2zkv2ycA5ifSSqoxOw%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fteams.microsoft.com%2fl%2fchannel%2f19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2fCAI%2520Accessibility%2520Support%2520Channel?groupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221198042%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=DCNVsZe8oDZWqm97AI0ee4OoWWecR6zw6qlhTuzP7MI%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2foutlook.office.com%2fbook%2fmicrosoft2779007%2540microsoft.onmicrosoft.com%2fs%2fbLeGBltMW0qI0WRLsTqx2A2?ismsaljsauthenabled&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221214666%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=CnCCdDgEmll2oUQUdMjXusAgV8PneqZVYvzgxAfk7Ik%3D&reserved=0). 
### Environment Details: 
Application Name: Visual Studio Code Services 
URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0)  
Chromium Edge: Version 144.0.3719.104 (Official build) dev (64-bit) 
Edition: Windows 11 Enterprise Insider Preview 
Version: 24H2 
OS build: 26558.1000 
### Repro steps: 

1. Turn on Screen reader and Hit the URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0) 
2. Tab navigate to Search edit filed and press enter on it. 
3. Start searching anything for example like python or selenium
4. Observe that Search Results or No Results Found Message Not Announced with Screen Reader. 

### Actual Result: 
When a user performs a search, the resulting status message (e.g., "Search Results" or "No Results Found") is visually displayed but is not announced by the screen reader. This leaves users unaware of the search outcome. 
### Expected Result: 
Status messages, such as "Search Results" or "No Results Found," should be programmatically exposed to assistive technologies and automatically announced by the screen reader without moving keyboard focus. 
### User Impact: 
Screen reader users may encounter challenges if the search results or the message indicating no results found are not announced by the screen reader. This can make it difficult for them to understand whether their search was successful or if no matching items were found, potentially impacting their ability to navigate and use the system effectively. 
### Attachment:

https://github.com/user-attachments/assets/afaeabae-9f1f-4988-81e5-49b4251fc511

https://github.com/user-attachments/assets/4477090f-a712-40d8-aff8-37d895cc50f6

**Comment by ntrogh** [comment]
@AnKushSingh05 Fixed on the website

**Comment by AnKushSingh05** [comment]
Hi @ntrogh 
I had verified issue in below environment, Issue is Partially Fixed. Only No of suggestions and No results found message issue is Fixed, But in Scan of mode list items are not being announced, this issue is not Fixed. Adding the attachments for reference.
[https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0)

https://github.com/user-attachments/assets/132a8d63-11fb-4b54-9bb2-ca7217b9d557
https://github.com/user-attachments/assets/17aec0dd-0a44-4ba0-ab8d-c7966c399913

**Comment by ntrogh** [comment]
@AnKushSingh05 The remaining issue should now be resolved on the website.
